app/routes/events/utils.js

```diff
@@ -63,6 +63,8 @@ const eventCreateAndUpdateFields = [
   'useCaptcha',
   'tags',
   'pools',
+  'registrationDeadlineHours',
+  'registrationCloseTime',
```

Comment:
`registrationCloseTime` is read-only in backend, and is not set by the edit/create form, so it should not be needed here.